examples/src/main/python/sql/arrow.py

```diff
@@ -32,8 +32,8 @@
 
 
 def dataframe_with_arrow_example(spark):
-    import numpy as np
-    import pandas as pd
+    import numpy as np  # type: ignore[import]
```

Comment:
Numpy has hinting as well, why ignore it? I got this working on https://github.com/zero323/pyspark-stubs/pull/464

Reply:
Because, AFAIK, released versions are not typed ‒ doesn't seem there were any released versions since https://github.com/numpy/numpy/commit/11b95d15f10c2bc652ed19d5e27efa0384396cb8.